src/doc/rustc/src/platform-support/riscv64gc-unknown-linux-gnu.md

```diff
@@ -0,0 +1,127 @@
+# `riscv64gc-unknown-linux-gnu`
+
+**Tier: 2 (with Host Tools)**
+
+RISC-V targets using the *RV64I* base instruction set with the *G* collection of extensions, as well as the *C* extension.
+
+
+## Target maintainers
+
+-  TODO
+
+
+## Requirements
+
+This target requires:
+
+* Linux Kernel version 4.20 or later
```

Comment:
We may need to raise this to 6.6 given https://github.com/rust-lang/rust/issues/117022#issuecomment-1841043719

Reply:
I'd leave that decision outside of a purely documentation PR.